package.json

```diff
@@ -14,7 +14,7 @@
     "wallet_demo": "./node_modules/.bin/parcel src/samples/wallet_demo.html --https --no-cache",
     "onboarding_demo": "./node_modules/.bin/parcel src/samples/onboarding_demo.html --https --no-cache",
     "typecheck": "./node_modules/.bin/tsc --noEmit",
-    "validate_lockfile": "./node_modules/.bin/lockfile-lint --type npm --path package-lock.json --validate-https --allowed-hosts npm",
+    "validate_lockfile": "./node_modules/.bin/lockfile-lint --type npm --path package-lock.json --allowed-hosts npm github.com --allowed-schemes \"https:\" \"git+https:\"",
```

Comment:
what is this change for?

Reply:
@SanchayPahalwani can you explain please

Reply:
@SanchayPahalwani

Reply:
This is package to verify all npm dependencies are coming from npm in the package-lock.json and wasn't passing the check for one of the strong pubsub package that's why github was also added as a valid host here